Kickstarter-iOS/Features/PaymentMethods/Controller/PaymentMethodsViewController.swift

```diff
@@ -145,6 +158,7 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
 
   @objc private func edit() {
     self.viewModel.inputs.editButtonTapped()
+    self.viewModel.inputs.shouldCancelPaymentSheetAppearance(state: true)
```

Comment:
Should we also call this if a user taps the `< Account` button in the nav bar?

Reply:
So good catch, if you find this to be the case in testing then yes we should do so.
However in my testing I didn't find that popping the page off the stack caused the in-flight payment sheet to come up. I'm assuming it's because the delegates are released and there are no more references to the payment sheet.